<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisualGPG</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>9</y>
      <width>591</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_my_certificates">
     <attribute name="title">
      <string>My certificates</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>581</width>
        <height>271</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>19</number>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>E-mail</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valid from</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valid until</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Key-ID</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_all_certificates">
     <attribute name="title">
      <string>All certificates</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetAllCerts">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>581</width>
        <height>271</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>19</number>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>E-mail</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valid from</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valid until</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Key-ID</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest1">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAboutGnuPG"/>
    <addaction name="actionAboutVisualGPG"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuTest1"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImportCertificates"/>
   <addaction name="actionLookupCertificate"/>
   <addaction name="actionRedisplayCertificates"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionImportCertificates">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icons/work-artfiles/load_certif.png</normaloff>:/icons/work-artfiles/load_certif.png</iconset>
   </property>
   <property name="text">
    <string>Import certificates</string>
   </property>
   <property name="toolTip">
    <string>Import certificates from files</string>
   </property>
  </action>
  <action name="actionLookupCertificate">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icons/work-artfiles/lookup_certif.png</normaloff>:/icons/work-artfiles/lookup_certif.png</iconset>
   </property>
   <property name="text">
    <string>Lookup certificates</string>
   </property>
   <property name="toolTip">
    <string>Import certificates from server</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAboutVisualGPG">
   <property name="text">
    <string>About VisualGPG</string>
   </property>
  </action>
  <action name="actionRedisplayCertificates">
   <property name="icon">
    <iconset>
     <normalon>:/icons/work-artfiles/reload_certif.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Redisplay Certificates</string>
   </property>
   <property name="toolTip">
    <string>Redisplay the certificates from GnuPG</string>
   </property>
  </action>
  <action name="actionAboutGnuPG">
   <property name="text">
    <string>About GnuPG (gpg2 --version)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
